tests/test_models.py

```diff
@@ -3177,7 +3177,7 @@ def test_replace_contributor(self):
     def test_permission_override_on_readded_contributor(self):
 
         # A child node created
-        self.child_node = NodeFactory(parent=self.project, creator=self.auth)
+        self.child_node = NodeFactory(parent=self.project, creator=self.auth.user)
```

Comment:
It was not failing on develop, but it was failing locally after these changes. `creator` is supposed to be a `User`, not an `Auth` object.

> `creator = fields.ForeignField('user', index=True)`
> in website/project/models.py:Node

Traceback from unmodified test:
```
======================================================================
ERROR: test_permission_override_on_readded_contributor (tests.test_models.TestProject)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/matt/Devel/osf.io/tests/test_models.py", line 3180, in test_permission_override_on_readded_contributor
    self.child_node = NodeFactory(parent=self.project, creator=self.auth)
  File "/Users/matt/.virtualenvs/osf/lib/python2.7/site-packages/factory/base.py", line 81, in __call__
    return cls.create(**kwargs)
  File "/Users/matt/.virtualenvs/osf/lib/python2.7/site-packages/factory/base.py", line 515, in create
    return cls._generate(True, attrs)
  File "/Users/matt/.virtualenvs/osf/lib/python2.7/site-packages/factory/base.py", line 440, in _generate
    obj = cls._prepare(create, **attrs)
  File "/Users/matt/.virtualenvs/osf/lib/python2.7/site-packages/factory/base.py", line 414, in _prepare
    return cls._create(target_class, *args, **kwargs)
  File "/Users/matt/Devel/osf.io/tests/factories.py", line 90, in _create
    instance.save()
  File "/Users/matt/Devel/osf.io/website/project/model.py", line 1599, in save
    auth_signals.node_created.send(self)
  File "/Users/matt/.virtualenvs/osf/lib/python2.7/site-packages/blinker/base.py", line 267, in send
    for receiver in self.receivers_for(sender)]
  File "/Users/matt/Devel/osf.io/website/project/mailing_list.py", line 29, in create_mailing_list_subscription
    subscription.add_user_to_subscription(node.creator, 'email_transactional', save=True)
  File "/Users/matt/Devel/osf.io/website/notifications/model.py", line 37, in add_user_to_subscription
    if self.owner._id not in user_subs.setdefault(user._id, []):
AttributeError: 'NoneType' object has no attribute '_id'
```